include/oneapi/dpl/internal/scan_by_segment_impl.h

```diff
@@ -105,20 +105,17 @@ struct __sycl_scan_by_segment_impl
     template <typename... _Name>
     using _SegScanPrefixPhase = __seg_scan_prefix_kernel<__is_inclusive, _Name...>;
 
-    template <typename _BackendTag, typename _ExecutionPolicy, typename _Range1, typename _Range2, typename _Range3,
-              typename _BinaryPredicate, typename _BinaryOperator, typename _T>
+    template <typename _BackendTag, typename _Range1, typename _Range2, typename _Range3, typename _BinaryPredicate,
+              typename _BinaryOperator, typename _T>
     void
-    operator()(_BackendTag, _ExecutionPolicy&& __exec, _Range1&& __keys, _Range2&& __values, _Range3&& __out_values,
-               _BinaryPredicate __binary_pred, _BinaryOperator __binary_op, _T __init, _T __identity)
+    operator()(oneapi::dpl::__internal::__device_backend_tag, sycl::queue& __q, _Range1&& __keys, _Range2&& __values,
```

Comment:
We are introducing an internal concrete tag name as hardcoded name.. It was not before.
BTW, why whether don't use overload by a hetero tag `__hetero_tag<_BackendTag>`?

Reply:
You point to someone of submitters.
All submitters which works with `sycl::queue` should be always parametrized by  `oneapi::dpl::__internal::__device_backend_tag` tag.

Reply:
Does `__device_backend_tag` mean SYCL backend?

Reply:
Yes, correct.

Reply:
@mmichel11 please note to remove the first parameter `oneapi::dpl::__internal::__device_backend_tag` from this function and it's calls when you will later move this submitter it into proper namespace.
Also it should be removed from: `__sycl_scan_by_segment_impl::operator()`, `__parallel_find_or_impl_one_wg::operator()`, `__parallel_find_or_impl_multiple_wgs::operator()`, `__parallel_transform_reduce_small_submitter::operator()`, `__parallel_transform_reduce_device_kernel_submitter::operator()`, `__parallel_transform_reduce_work_group_kernel_submitter::operator()`

Reply:
Issue created: https://github.com/uxlfoundation/oneDPL/issues/2211